### PR TITLE
Modified files: moved namespace deletion check to their respective main files. 

### PR DESCRIPTION
**Pull request by vibhor995**
Signed-off-by: vibhor995 <vvibhr995@gmail.com>

**Removed deletion check from cleanup files and added it to the main playbook in always block**:

1) It fixes the effort of manually deletion of namespaces if playbook failed when any of the task having 
     some errors.
2) Bring back the cluster to  its default state  even if any task is failing ,as namespaces is associated to the deployement gets deleted after complete run of playbook.

@ksatchit  @dargasudarshan @gprasath

### CODE REVIEW

**Comment by ksatchit** [comment]
@vibhor995 , how do we plan to handle namespaces stuck in "terminating" state post delete. I dont see confirmation checks for the namespace deletion.. this would be a good addition. And how about converting this into a namespace deletion util :)

**Comment by vibhor995** [comment]
@ksatchit  yes we can do that ...i am working on it

**Comment by vibhor995** [comment]
@ksatchit  review please

File: e2e/ansible/playbooks/utils/delete_namespaces.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- name: Check the namespace status.
+  shell: source ~/.profile; kubectl get ns 
+  args:
+    executable: /bin/bash
+  register: np
+  delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+  until: "'{{ namespace }}' not in np.stdout"
+  delay: 30
+  retries: 15
+  ignore_erros: true
```
Comment (ksatchit) [inline e2e/ansible/playbooks/utils/delete_namespaces.yml]: Typo at "ignore_erros" ! Also, why are we ignoring this ? The idea is to make an effort at all costs. 

Also, the namespace should ideally be passed as a param to util as a best practice (though it should work even w/o by virtue of adding vars file)

**Comment by ksatchit** [comment]
Also, please resolve merge conflicts

**Comment by gprasath** [comment]
@vibhor995 Can you execute all the tests with your changes and check if there is any impact.

**Pull request closed without merging**